test/e2e/specs/plans/plans__legacy-renew.ts

```diff
@@ -53,7 +53,9 @@ describe( DataHelper.createSuiteTitle( 'Plans (Legacy): Renew' ), function () {
 			await individualPurchasesPage.validatePurchaseTitle( planName );
 		} );
 
-		it( 'Renew plan', async function () {
+		// causing failures on trunk
```

Comment:
Could I ask why this step being disabled? Is this feature behind a feature flag and not enabled by default on trunk? 

If this step has to be skipped, is there a followup issue or task filed to re-enable it?

Reply:
In addition, I suspect that disabling this step will also result in rest of the spec no longer function as expected.

Reply:
This is correct, so I've disabled the remaining steps. I've pinged you in slack with the details for discussion.

Reply:
Basically, this test has been failing on trunk for all patches and is preventing merges. See https://teamcity.a8c.com/project.html?projectId=calypso_WebApp&testNameId=-2979286403470822990&tab=testDetails

Reply:
> This is correct, so I've disabled the remaining steps. I've pinged you in slack with the details for discussion.
I haven't received any pings on Slack, but from what I can tell there is no failure on trunk in Pre-Release Tests. Are you sure you're looking at the correct build?